src/test-utils/mocks/mockGqlResponse.ts

```diff
@@ -0,0 +1,59 @@
+const mockGqlResponse = {
+  CustomerConversationsList: {
+    conversations: [
+      {
+        id: 1,
+        customer: {
+          displayName: "Jane Doe",
+          __typename: "Customer",
+        },
+        messages: [
+          {
+            content: {
+              text: "world",
+              type: "text",
+            },
+            __typename: "Message",
+          },
+          {
+            content: {
+              text: "hello",
+              type: "text",
+            },
+            __typename: "Message",
+          },
+        ],
+        __typename: "Conversation",
+      },
+      {
+        id: 2,
+        customer: {
+          displayName: "Chan Tai Man",
+          __typename: "Customer",
+        },
+        messages: [
+          {
+            content: {
+              type: "image",
+              mediaUrl:
+                "https://media.smooch.io/apps/60014840c2cbff000c54f83f/conversations/472d5a7954f69b57b549e6c5/lcOwivrqBOrL2B9NtPj6wCfI/Vlt4uiF5TGxJVrk5txIIG0BA.jpeg",
```

Comment:
Better to use dummy URLs and not expose the _real_ appId/conversationId
Not a huge deal since it's a staging environment and the secret key is not exposed, but doesn't cost us anything to avoid it

Reply:
Also, if multiple tests depend on a single mock, it'll end up being edited for one test, and making another test fail (I noticed you referenced the test `"world"` directly in a test above)

Reply:
Some other tests also check for the count of items. So if I modify this (e.g for my own test), it makes your tests above fail